lib/components/user/monitored-trip/saved-trip-screen.js

```diff
@@ -144,11 +150,22 @@ class SavedTripScreen extends Component {
         .notOneOf(otherTripNames, 'Another saved trip already uses this name. Please choose a different name.')
       const validationSchema = yup.object(clonedSchemaShape)
 
+      // Create a modified copy of monitoredTrip for editing.
+      const clonedMonitoredTrip = clone(monitoredTrip)
+
+      // To spare users the complexity of the departure/arrival delay thresholds,
+      // we create an overall delayThreshold field for UI editing, initialized as the smallest
+      // between 'arrivalVarianceMinutesThreshold' and 'departureVarianceMinutesThreshold'.
+      clonedMonitoredTrip.delayThreshold = Math.min(
+        monitoredTrip.arrivalVarianceMinutesThreshold,
+        monitoredTrip.departureVarianceMinutesThreshold
+      )
```

Comment:
Maybe all of this logic (cloning and setting delay field) should happen in a method `prepareTripForEditing`? Alternatively, if there's a way to just set both of these values using Formik (rather than manipulating the object), that might be preferable.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/323/commits/814d9df6102f1c34cf1fedb8c45ba57bb3f24233.